### PR TITLE
4.6 docs (+ a few missing 4.5 docs)

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -3100,6 +3100,32 @@ Skip Server Certificate Verification
...
++-------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"LoginButtonBorderColor": ""`` with string input.                                 |
++-------------------------------------------------------------------------------------------------------------------------------+
+
+Login Button Text Color
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Specify the color of the email login button text for white labeling purposes.
+
++-------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"LoginButtonTextColor": ""`` with string input.                                   |
```
Comment (esethna) [inline source/administration/config-settings.rst]: What type of string input is accepted (ie what format should the color be). If hex code, does it need a # before the hex code?

Comment (esethna) [reply source/administration/config-settings.rst]: Comment applies to all color inputs

File: source/help/messaging/keyboard-shortcuts.rst [hunk 2]
```diff
@@ -27,6 +27,8 @@ Navigation
 +----------------------------------------+----------------------------------------+------------------------------------------------------------------------------+
 | CTRL+SHIFT+M                           | CMD+SHIFT+M                            | Open recent mentions                                                         |
 +----------------------------------------+----------------------------------------+------------------------------------------------------------------------------+
+| CTRL+SHIFT+L                           | CMD+SHIFT+L                            | Set focus to input field                                                     |
```
Comment (esethna) [inline source/help/messaging/keyboard-shortcuts.rst]: 0/5, Do we need to specify center channel only (not RHS)?

**Comment by jasonblais** [comment]
@esethna Updated based on feedback, thanks!

**Pull request merged**